llama_hub/airbyte_cdk/base.py

```diff
@@ -0,0 +1,33 @@
+from typing import Any, List, Mapping, Optional
+
+from llama_index.readers.base import BaseReader
+from llama_index.readers.schema.base import Document
+from airbyte_protocol.models.airbyte_protocol import AirbyteRecordMessage
+from airbyte_cdk.sources.embedded.base_integration import BaseEmbeddedIntegration
+from airbyte_cdk.sources.embedded.runner import CDKRunner
+
+
+class AirbyteCDKReader(BaseReader, BaseEmbeddedIntegration):
+    """AirbyteCDKReader reader.
+
+    Retrieve documents from an Airbyte source implemented using the CDK.
+
+    Args:
+        source_class: The Airbyte source class.
+        config: The config object for the Airbyte source.
+    """
+
+    def __init__(
+        self,
+        source_class: Any,
+        config: Mapping[str, Any],
+    ) -> None:
+        """Initialize with parameters."""
+
+        super().__init__(config=config, runner=CDKRunner(source=source_class(), name=source_class.__name__))
```

Comment:
Left the same comment on the langchain equivalent, but we could validate the config via spec at this stage here as well. Again, probably something to handle on the CDKRunner end.

Reply:
Yeah, I'll tackle this one on the CDK side